tests/integration-tests/src/appinventory.rs

```diff
@@ -0,0 +1,204 @@
+use std::{env, path::Path};
+
+use duct::cmd;
+use reqwest::Url;
+use serde::{Deserialize, Serialize};
+use std::fs::File;
+use std::io::{BufReader, Cursor};
+use std::path::PathBuf;
+use tar::Archive;
+use tempfile::TempDir;
+use which::which;
+
+use crate::{run_command, twoliter_build::copy_project_to_temp_dir, TWOLITER_PATH};
+
+#[derive(Serialize, Deserialize)]
+pub struct GithubRelease {
+    tag_name: String,
+}
+
+async fn find_latest_version(repository: &str) -> String {
+    // We make a get request to fetch the github releases of twoliter
+    let url = Url::parse(&format!(
+        "https://api.github.com/repos/bottlerocket-os/{repository}/releases"
```

Comment:
Can we just use https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#get-the-latest-release to get the latest? Should be fine as long as we don't mark release candidates in Twoliter as "latest"